nss-day-cohort-43/nuttyshell-the-real-housewives-of-transylvania
# Jt nutshell public chat edit

# Description
Created an edit button attached to all messages that were sent by the current active user. Once clicked, it allows you to change any portion of the previous message. You may then save your edit, and the page will refresh to show your newly edited message.
Fixes # (issue)
## Type of change



# Testing Instructions
1.) login or register to new account
2.) send a new message, or view a message you previously sent on the logged in account's username
3.) You should see a pencil icon for editing and a wastepaper basket for delete under your own messages
4.) click the pencil, which should then change your old message into an input box. 
5.) in the input box, type out a newly edit message and hit save
6.) message should refresh and display your edited message.
# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
